src/pocketmine/item/RawChicken.php

```diff
@@ -40,6 +40,7 @@ public function getAdditionalEffects() : array{
 		if(mt_rand(0, 9) < 3){
 			return Effect::getEffect(Effect::HUNGER)->setDuration(600);
 		}
```

Comment:
It should be an array to conform with the Food declaration. https://github.com/pmmp/PocketMine-MP/blob/master/src/pocketmine/item/Food.php#L50